### PR TITLE
add warning when empty or broken mail address is entered

### PR DESCRIPTION
**Pull request by jospoortvliet**
I see a lot of empty mail address fields entered; and this should help fix it.
@blizzz made all errors now look pretty - that is, show the nice header etc.
Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by jospoortvliet** [comment]
Every page has been tested locally and works.

File: page-architecture-whitepapersubmit.php [hunk 1]
```diff
@@ -1,47 +1,88 @@
...
+
 <?php
 if(isset($_POST['email'])) {
-   function died($error) {
-    // error code goes here
-        echo "We are very sorry, but there were error(s) found with the form you submitted: ";
-        echo "<br />";
-        echo $error."<br />";
-        echo "Please go back and fix these errors.<br />";
-        die();
```
Comment (blizzz) [inline page-architecture-whitepapersubmit.php]: this ``die`` caused WP to not load further resources and thus the output could not be rendered properly.

**Comment by blizzz** [comment]
self-approving is sorta stupid, ideally someone else has another look at this spaghetti :)

File: page-architecture-whitepapersubmit.php [hunk 2]
```diff
@@ -64,39 +105,14 @@ function clean_string($string) {
...
-        <p>The whitepaper has been sent to <?php echo htmlentities($email_to) ?>,</p>
-        <p>check your spam folder if you can not find it!</p>
-    </div>
-</section>
-<?php
+    <section class="section--whitepaper">
+        <div class="container text-center">
+            <h3>Thank you for your interest in our whitepaper</h3>
+            <p>The whitepaper has been sent to <?php echo($email_to); ?>,</p>
+            <p>check your spam folder if you can not find it!</p>
```
Comment (NinaNC) [inline page-architecture-whitepapersubmit.php]: cannot

File: page-architecture-whitepapersubmit.php [hunk 3]
```diff
@@ -1,47 +1,88 @@
...
+		<section class="section--whitepaper">
+			<div class="container text-center">
+				<h3>Sorry, there was an error with the form you submitted</h3>
+				<p>The error(s) detected:<br/>
+					<?php
+					echo $error . "<br />";
+					?>
+				</p>
+				<p>Use the back key to go to the previous page and fix the
+					problem!</p>
```
Comment (NinaNC) [inline page-architecture-whitepapersubmit.php]: Maybe "Go back to the previous page to fix the issue!"?

File: page-storage-whitepapersubmit.php [hunk 4]
```diff
@@ -1,90 +1,107 @@
...
-			<div class="col-md-6 topheader">
-				<h1><?php echo $l->t('Secure access to your data anywhere');?></h1>
-				<h2><?php echo $l->t('Avoid migration and access your files wherever they are stored');?></p>
-			</div>
+	<!-- success html here -->
+	<section class="section--whitepaper">
+		<div class="container text-center">
+			<h3>Thank you for your interest in our data sheets</h3>
+			<p>The data sheets have been sent to <?php echo($email_to); ?>,</p>
+			<p>check your spam folder if you can not find it!</p>
```
Comment (NinaNC) [inline page-storage-whitepapersubmit.php]: Capital C for Check? I don't know if that makes sense in the page

**Pull request merged**